WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -49,15 +49,51 @@ public void onErrorResponse(VolleyError volleyError) {
         WordPress.getRestClientUtilsV1_1().get(path, listener, errorListener);
     }
 
-    private static List<Person> peopleListFromJSON(JSONArray jsonArray, int localTableBlogId) {
+    public static void updateRole(final String siteID, String userID, String newRole, final int localTableBlogId, final UpdateUserCallback callback) {
+        com.wordpress.rest.RestRequest.Listener listener = new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject jsonObject) {
+                if (jsonObject != null && callback != null) {
+                    Person person = Person.fromJSON(jsonObject, siteID, localTableBlogId);
+                    callback.onSuccess(person);
```

Comment:
I've fixed the `updateRole` method. However, `fetchUsers()` is already handling the `null` case gracefully. It shouldn't happen, but if the mapped `Person` models are all `null` it'll just be an empty list and we'll refresh it in `PeopleManagementActivity`. I don't think any changes are needed here.